crates/humanode-runtime/src/weights/mod.rs

```diff
@@ -1,4 +1,4 @@
-// DO NOT EDIT! This file is generated by "utils/weights/benchmark-all" command.
+// DO NOT EDIT! This file is generated by "./utils/weights/benchmark-all" command.
```

Comment:
This is interesting, we probably should make sure this doesn't change in this PR.

Reply:
https://github.com/humanode-network/humanode/pull/861/commits/e1177f01591ccdd5b9d469692d2241578dfb880a